src/config/loadGitHubAppsFromEnvironment.ts

```diff
@@ -58,12 +63,32 @@ export class GitHubAppsRegistry {
 }
 
 export function loadGitHubAppsFromEnvironment(env) {
-  const apps = new GitHubAppsRegistry();
+  const configs = new GitHubAppsConfigHelper();
+  let config;
 
   if (env.GH_APP_IDENTIFIER && env.GH_APP_SECRET_KEY) {
-    const org = '__tmp_org_placeholder__';
-    apps.register(new GitHubApp(org, env));
+    // Collect config by (proleptic) envvar number. Once we have GH_APP_1_FOO
+    // this will make more sense. We'll collect stuff in config and then
+    // instantiate a GitHubApp once all config has been collected for each
+    // (once we've made a full pass through process.env).
+
+    config = configs.forNumber(1);
+    config.org = '__tmp_org_placeholder__';
+    config.auth = {
+      appId: Number(env.GH_APP_IDENTIFIER),
+      privateKey: env.GH_APP_SECRET_KEY.replace(/\\n/g, '\n'),
+    };
+    config.project = {
+      node_id: env.ISSUES_PROJECT_NODE_ID || 'PVT_kwDOABVQ184AOGW8',
```

Comment:
Maybe we should store the sentital values (which I assume are for our default repo from before) in a single obj or set of consts somewhere, so it's clear what we are defaulting to?

Reply:
They're removed on the next PR (#524). I've been preserving this `|| 'PVT_foo'` pattern temporarily, this is the last place we have it.